docs/index.md

```diff
@@ -120,6 +120,20 @@ render((
 
 > Note: If there are fields in the `formData` that are not represented in the schema, they will be retained by default. If you would like to remove those extra values on form submission, then set the `omitExtraData` prop to `true`. Set the `liveOmit` prop to true in order to remove extra data upon form data change.
 
+#### Prevent early form submission (via enter key press)
```

Comment:
thanks -- would it be possible to add a test?

Reply:
I couldn't figure out how to get the test to show the default behavior, i.e. the submit event fired when pressing the enter key. My PR description has a bit of the code I was tried that didn't work (submit function was never called).